src/domain/event/EventPageContainer.tsx

```diff
@@ -53,7 +53,9 @@ const EventPageContainer: React.FC = () => {
               {/* Show event content only if event is open */}
               {!eventClosed && <EventContent event={event} />}
               {/* Hide similar event on SSR to make initial load faster */}
-              {isClient && <SimilarEvents event={event} />}
+              {isClient && getFeatureFlags().SHOW_SIMILAR_EVENTS && (
```

Comment:
Minor readability improvement would be to call this function in top of this function and destructure SHOW_SIMILAR_EVENTS. But that is debatable :D

Reply:
fixed